client/src/app.tsx

```diff
@@ -233,11 +234,25 @@ export const client = new ApolloClient({
   link: ApolloLink.from(links)
 });
 
+// add action to all snackbars
+const notistackRef = React.createRef<SnackbarProvider>();
```

Comment:
I find it strange to initialize refs outside of component scope, but why not

Reply:
I used the examples from the doc!